backend/drm/drm-properties.h

```diff
@@ -56,4 +56,7 @@ bool wlr_drm_get_connector_props(int fd, uint32_t id, union wlr_drm_connector_pr
 bool wlr_drm_get_crtc_props(int fd, uint32_t id, union wlr_drm_crtc_props *out);
 bool wlr_drm_get_plane_props(int fd, uint32_t id, union wlr_drm_plane_props *out);
 
+bool wlr_drm_get_prop(int fd, uint32_t obj, uint32_t prop, uint64_t *ret);
+void *wlr_drm_get_prop_blob(int fd, uint32_t obj, uint32_t prop, size_t *ret_len);
```

Comment:
I agree that the union looks strange in the header (most uses of unions usually raise a few eyebrows), but due to anonymous structs/unions, people don't even need to think of that detail; it just looks like using a struct in the code. It gives a nice, safe, clean-looking interface, and overall I don't think it's particularly 'offensive'.

Also do we even want to expose the wlr_drm_{crtc,plane} types directly to users? I not really sure how useful they would be.